src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx

```diff
@@ -278,7 +286,12 @@ const PyodideRunner = () => {
   };
 
   return (
-    <div className={`pythonrunner-container`}>
+    <div
+      className={`pythonrunner-container pyodiderunner${
+        active ? " pyodiderunner--active" : ""
+      }`}
+      style={{ display: showRunner ? "flex" : "none" }}
+    >
```

Comment:
definitely feel like I'm missing something with the difference between `active` and `showRunner` but it feels like they should be one variable. Unless its essential having one as a prop and another as a state variable feels like they might get out of sync

Reply:
I guess the point is that we want them out of sync at a certain point, which is when the user's code changes to mean that a switch is necessary, but they haven't clicked run yet. The reason there are two variables is so it can switch runners when the code changes without changing the DOM straightaway. So `active` means that's the runner that will actually run the code on the next code run, whereas `showRunner` means that' s the runner the user sees in the DOM.

Reply:
Gotcha, ok that makes sense in terms of the styling 👍